app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -99,277 +133,323 @@ fun CreateActivityScreen(
         )
       },
       content = { paddingValues ->
-        Column(
-            modifier =
-                Modifier.padding(paddingValues)
-                    .fillMaxSize()
-                    .verticalScroll(scrollState)
-                    // .background(color = Color(0xFFFFFFFF))
-                    .testTag("activityCreateScreen"),
-        ) {
-          // Carousel()
-          Spacer(modifier = Modifier.height(8.dp))
-          OutlinedTextField(
-              value = title,
-              onValueChange = { title = it },
-              label = { Text("Title") },
-              modifier = Modifier.padding(8.dp).fillMaxWidth().testTag("inputTitleCreate"),
-              placeholder = { Text(text = stringResource(id = R.string.request_activity_title)) },
-          )
-          Spacer(modifier = Modifier.height(8.dp))
+        if (isCamOpen) {
+          Box(modifier = Modifier.fillMaxSize().padding(paddingValues)) {
+            CameraPreview(controller, Modifier.fillMaxSize())
+            IconButton(
+                onClick = { isCamOpen = false }, modifier = Modifier.align(Alignment.TopEnd)) {
+                  Icon(Icons.Default.ArrowBack, contentDescription = "Close camera")
+                }
+            IconButton(
+                onClick = {
+                  takePhoto(
+                      controller,
+                      { bitmap ->
+                        items += bitmap
```

Comment:
when a photo is taken, it is added to the list of items and the camera is closed